ApiUtil.ts

```diff
@@ -25,4 +25,12 @@ export function getAtlasList(): Promise<WPAtlas[]> {
     return fetcher(postUrl);
 }
 
+export async function getStaticContent(slugs: string[])
```

Comment:
more descriptive name?

Reply:
hmm, what should we name it? `getStaticContentFromWP`?

@alisman any suggestions?